predicators/ground_truth_models/doors/options.py

```diff
@@ -293,3 +294,233 @@ def _get_position_in_doorway(cls, room: Object, door: Object,
             ty = (y0 + y1) / 2
             position_cache[(room, door)] = (tx, ty)
         return position_cache[(room, door)]
+
+
+class DoorknobsGroundTruthOptionFactory(DoorsGroundTruthOptionFactory):
```

Comment:
ditto w.r.t comment about ground truth nsrts factory; please look at the ground truth options factory for the `cover` env and try to use that pattern to minimize code duplication